lib/remote_system.rb

```diff
@@ -150,8 +150,12 @@ def check_retrieve_files_dependencies
   end
 
   # Reads a file from the System. Returns nil if it does not exist.
-  def read_file(file)
-    run_command("cat", file, stdout: :capture, privileged: true)
+  def read_file(file, options = {})
+    if options[:privileged]
+      run_command("cat", file, stdout: :capture, privileged: true)
```

Comment:
Not exposing the cheetah options makes sense, but that could also be circumvented using

```ruby
command_options = {
  stdout: :capture,
  privileged: options.fetch(:privileged, false)
}
```
for example.

Having to adapt the tests is a relatively weak argument against clearer and more DRY code  imo ;)